hildon/hildon-gtk.h

```diff
@@ -18,6 +18,55 @@
 #define                                         __HILDON_GTK_H__
 
 #include                                        <gtk/gtk.h>
+/// Temporary hack to provide the types from the Maemo fork of gtk
+/// This was copy-pasted from hildonize-gtk-enums.diff from maemo-leste/gtk
```

Comment:
Pull requests should be complete.
Please try to avoid hacks, temporary hacks, dirty hacks and so on in your pull requests.
If this part of file is going to be changed please implement the needed changes in your pull request.
If it's not going to be changed, then remove this comment.

Reply:
This used to be a temporary hack while this was only a partial fix. But you reminded me that I had intended to move it to a separate file, which I've just done.